src/core/components/courses/TraineeFollowUpTable.vue

```diff
@@ -105,6 +106,9 @@ export default {
 
       return ability.can('read', 'learner_info');
     },
+    visibleColumns () {
+      return this.isBlended ? this.columns : this.columns.filter(column => column.name !== 'isConnected');
+    },
```

Comment:
Tu peux faire : `style: this.isBlended ? '' : 'display: none'`, sur la column isConnected directement, non ?

Reply:
Je me demande si comme habituellement on utilise la props visibleColumns de quasar, la bonne pratique ne serait pas de passer par cette props ici aussi. Ici on pourrait avoir une props visibleColumns pour traineeFollowupTable qui par defaut prend tous les noms de colonnes du tableau.
Vous en pensez quoi ?

Reply:
trop top ! je ne me souvenais pas de cette solution mais c'est x100 mieux